README.md

```diff
@@ -53,12 +53,7 @@ has some particular value set. Perhaps `:local? true` or `:endpoint "localhost"`
 
 ### Caveats
 
-Datomic Cloud's query supports a subset of the functionality of Datomic On-Prem.
```

Comment:
I think I'd rather leave this message in, or some portion of it at least. There are still a number of things that this library does not support (e.g., return maps, tuple, attr-preds, entity specs).